library/Makefile

```diff
@@ -318,6 +319,13 @@ version_features.c:
 	echo "  Gen   $@"
 	$(PERL) ../scripts/generate_features.pl
 
+psa_crypto_driver_wrappers.c: ../scripts/generate_driver_wrappers.py
+psa_crypto_driver_wrappers.c: ../scripts/data_files/driver_templates/psa_crypto_driver_wrappers.conf
+psa_crypto_driver_wrappers.c:
```

Comment:
https://github.com/ARMmbed/mbedtls/pull/5193/files/947cf611f2f935701b2cef3333532b1c325e02bc#r772816254
Refer above comment.
If the user is doing either
(1) replacing the whole ....driver_wrapper.c or patching it, the user should do the same with the template file.  I see no problem in that approach.
(2)I understand it could be hard to visualize what the template file would look like after some templates are added in for each entry point. For that I recommend referring the prototype(prototype/driver_wrapper_sample.conf) at https://github.com/ARMmbed/mbedtls/pull/5067/commits/e705d07021f0aaebf8604651d24ff836975ca28e